frontend/src/utilities/files.ts

```diff
@@ -11,6 +8,14 @@ export function download(filename: string, fileData: string): void {
 	element.click();
 }
 
+export function download(filename: string, fileData: string): void {
+	const type = filename.endsWith(".svg") ? "image/svg+xml;charset=utf-8" : "text/plain;charset=utf-8";
+	const blob = new Blob([fileData], { type });
+	const url = URL.createObjectURL(blob);
```

Comment:
I think we're currently forgetting to clean up with `URL.revokeObjectURL(url);`.